subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -395,6 +419,7 @@ uint8_t ull_cp_feature_exchange(struct ll_conn *conn)
 
 	return BT_HCI_ERR_SUCCESS;
 }
+#endif /* CONFIG_BT_CENTRAL) || CONFIG_BT_CTLR_SLAVE_FEAT_REQ */
```

Comment:
Woops :o) you're too generous with ")"s in comment

Reply:
Pokey pokey